cdk/src/open-data-platform/data-plane/data-import/handler-factory.ts

```diff
@@ -1,22 +1,25 @@
 import { APIGatewayProxyResult } from 'aws-lambda';
 import * as AWS from 'aws-sdk';
+import { ParallelSink } from './utils/parallel-sink';
+import { offset, limit } from './utils/stream-utils';
 
 // These libraries don't have types, so they are imported in a different way.
 const { chain } = require('stream-chain');
 const { streamArray } = require('stream-json/streamers/StreamArray');
 const Batch = require('stream-json/utils/Batch');
 const Pick = require('stream-json/filters/Pick');
 
+const SINK_CONCURRENCY = 5;
```

Comment:
I picked it somewhat arbitrarily because at this rate, the water-systems import happens in ~1 min. 

I guess what I'm trying to constrain is "write load for Aurora", which I see as concurrency x batchSize. Thinking about it some more, would you prefer if this was also a variable (like batch size) that could be set by the caller?

Reply:
Nope. Just leave a comment with that reason so people aren't confused by a magic number.

Reply:
Done :)